ui/src/app/index.js

```diff
@@ -3,7 +3,7 @@
 angular.module('bassa', ['ngAnimate', 'ngCookies', 'ngTouch',
   'ngSanitize', 'ui.router', 'ngMaterial', 'nvd3', 'app'])
 
-  .value('BassaUrl', 'http://localhost:5000')
+  .value('BassaUrl', 'https://192.168.64.9')
```

Comment:
For the web to contact API server through ingress

Reply:
This will break out existing APIs as a lot of APIs using `BassaUrl` value

Reply:
@vivonk bro, true but not sure how to handle this. Either we should manually change this to production Bassa API url when have deployed completely. Or we should use environment variables. I am unable to use environment variables here.

Reply:
Is there anyway to use a config file?

Reply:
@JaDogg bro true, we can use a config file. On total I feel we have two options
1. Get values using environment variables (this makes the deployment very easy as we can feasibly change API url anytime without rebuilding the image) 
2. Use a common config file. We will mount that rather copying it into container, doing this will result in similar benefit as of using env variables. But we might need to have an extra volume to store the file. 
@JaDogg @vivonk @agentmilindu any thoughts about this?

Reply:
Get value from somewhere and default to localhost?